src/torchaudio/_internal/module_utils.py

```diff
@@ -97,7 +97,8 @@ def decorator(func):
     {func.__doc__}
     """
 
-        UNSUPPORTED.append(wrapped)
+        if 'sox' not in func.__module__:
+            UNSUPPORTED.append(wrapped)
```

Comment:
Can you please add a comment saying this is a very temporary hack that we'll have to remove as soon as we remove the sox-related functionalities